src/JShrink/Minifier.php

```diff
@@ -32,102 +35,93 @@
 class Minifier
 {
     /**
-     * The input javascript to be minified.
+     * Contains the default options for minification. This array is merged with
+     * the one passed in by the user to create the request specific set of
+     * options (stored in the $options attribute).
      *
-     * @var string
+     * @var bool[]
      */
-    protected $input;
+    protected static array $defaultOptions = ['flaggedComments' => true];
 
-    /**
-     * Length of input javascript.
-     *
-     * @var int
-     */
-    protected $len = 0;
+    /** The input javascript to be minified. */
```

Comment:
How come removing the `@var` docblock comments? Would need to change from `@var int` to `@var string` if changing this from an integer to a string, but docblock comments are still useful for various annotation tools that utilise docblocks. Better to keep such comments in the codebase, IMO.

Reply:
I try to remove the `@var` annotation if the type is precisely defined directly in the property or argument. Documentation tools can use it because it is the native behavior of PHP.

If we support PHP 7.1+, then I will add `@var` annotation again.

Reply:
Ah, okay. Sounds good. :-)